--- a/src/main/java/com/misterblusky9/pocket/mixin/create/ScaledSubLevelSeatPositionMixin.java
+++ b/src/main/java/com/misterblusky9/pocket/mixin/create/ScaledSubLevelSeatPositionMixin.java
@@ -1,7 +1,6 @@
 package com.misterblusky9.pocket.mixin.create;
 
 import com.misterblusky9.pocket.PocketSized;
-import com.misterblusky9.pocket.config.PocketServerConfig;
 import com.simibubi.create.content.contraptions.actors.seat.SeatEntity;
 import dev.ryanhcode.sable.Sable;
 import dev.ryanhcode.sable.sublevel.SubLevel;
@@ -28,7 +27,6 @@
             final Entity.MoveFunction callback,
             final CallbackInfo ci
     ) {
-        if (!PocketServerConfig.scalePlayerInShrunkenSeat()) return;
         if (!(passenger instanceof Player)) return;
 
         final SeatEntity seat = (SeatEntity) (Object) this;
